VirtualList/tests/VirtualList-translate-specs.js

```diff
@@ -0,0 +1,811 @@
+import '@testing-library/jest-dom';
+import {act, fireEvent, render, screen} from '@testing-library/react';
+
+import Item from '../../Item';
+import VirtualList from '../VirtualList';
+
+describe('VirtualList translate', () => {
+	let
+		clientSize,
+		dataSize,
+		getScrollTo,
+		handlerOnScroll,
+		handlerOnScrollStart,
+		handlerOnScrollStop,
+		items,
+		itemSize,
+		myScrollTo,
+		onScrollCount,
+		onScrollStartCount,
+		onScrollStopCount,
+		renderItem,
+		resultScrollLeft,
+		resultScrollTop,
+		startScrollTop;
+
+	beforeEach(() => {
+		clientSize = {clientWidth: 1280, clientHeight: 720};
+		dataSize = 200;
+		items = [];
+		itemSize = 60;
+		onScrollCount = 0;
+		onScrollStartCount = 0;
+		onScrollStopCount = 0;
+		resultScrollLeft = 0;
+		resultScrollTop = 0;
+		startScrollTop = 0;
+
+		getScrollTo = (scrollTo) => {
+			myScrollTo = scrollTo;
+		};
+		handlerOnScroll = () => {
+			onScrollCount++;
+		};
+		handlerOnScrollStart = (e) => {
+			startScrollTop = e.scrollTop;
+			onScrollStartCount++;
+		};
+		handlerOnScrollStop = (done, testCase) => (e) => {
+			onScrollStopCount++;
+			resultScrollLeft = e.scrollLeft;
+			resultScrollTop = e.scrollTop;
+
+			testCase();
+			done();
+		};
+		renderItem = ({index, ...rest}) => { // eslint-disable-line enact/display-name
+			return (
+				<Item {...rest}>
+					{items[index].name}
+				</Item>
+			);
+		};
+
+		for (let i = 0; i < dataSize; i++) {
+			items.push({name: 'Account ' + i});
+		}
+	});
+
+	afterEach(() => {
+		clientSize = null;
+		dataSize = null;
+		getScrollTo = null;
+		handlerOnScroll = null;
+		handlerOnScrollStart = null;
+		handlerOnScrollStop = null;
+		items = null;
+		itemSize = null;
+		myScrollTo = null;
+		onScrollCount = null;
+		onScrollStartCount = null;
+		onScrollStopCount = null;
+		renderItem = null;
+		resultScrollLeft = null;
+		resultScrollTop = null;
+		startScrollTop = null;
+	});
+
+	test('should render a list of \'items\'', () => {
+		render(
+			<VirtualList
+				clientSize={clientSize}
+				dataSize={dataSize}
+				itemRenderer={renderItem}
+				itemSize={itemSize}
+				scrollMode="translate"
+			/>
+		);
+
+		const expected = 'Account 0';
+		const actual = screen.getByRole('list').children.item(0).textContent;
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should render overhang items when clientSize is not given', () => {
```

Comment:
😕

Reply:
I fixed it to

test('should render overhang items  when \'clientSize\' and outer DOM size are not specified', () => {